### PR TITLE
[zh-cn] synced from #10b7de7 to #ef4a046

### PR DESCRIPTION
**Pull request by Jinjiang**
ref: `/docs/en/*` parts of https://github.com/vuejs/vue-router/compare/10b7de7...ef4a046

/ping @Justineo

### CODE REVIEW

**Comment by Jinjiang** [comment]
All translated @Justineo

File: docs/zh-cn/essentials/getting-started.md [hunk 1]
```diff
@@ -63,7 +63,29 @@ const app = new Vue({
 // 现在，应用已经启动了！
 ```
 
-你可以看看这个例子
-[live](https://jsfiddle.net/yyx990803/xgrjzsup/)。
+通过注入路有，我们可以用 `this.$router` 来访问它，就像在任何组件里用 `this.$router` 访问当前路有一样。
```
Comment (fnlctrl) [inline docs/zh-cn/essentials/getting-started.md]: 路有 => 路由

File: docs/zh-cn/essentials/getting-started.md [hunk 2]
```diff
@@ -63,7 +63,29 @@ const app = new Vue({
...
+    goBack () {
+      window.history.length > 1
+        ? this.$router.go(-1)
+        : this.$router.push('/')
+    }
+  }
+}
+```
+
+该文档通篇都常使用 `router` 实例。留意一下 `this.$router` 和 `router` 使用起来完全一样。我们使用 `this.$router` 的原因是我们并不想在每个独立需要封装路有的组件中都导入路由。
```
Comment (fnlctrl) [inline docs/zh-cn/essentials/getting-started.md]: 路有 => 路由

File: docs/zh-cn/essentials/getting-started.md [hunk 3]
```diff
@@ -63,7 +63,29 @@ const app = new Vue({
...
+
+你可以看看这个[现场的](https://jsfiddle.net/yyx990803/xgrjzsup/)例子。
```
Comment (fnlctrl) [inline docs/zh-cn/essentials/getting-started.md]: live可能翻译成“在线"更常见一些

**Review by fnlctrl — APPROVED** [review]

**Pull request merged**